scripts/taco_to_burrito.py

```diff
@@ -30,33 +34,51 @@ def main():
     for file in os.scandir(source_path):
         if(file.name.endswith(".xml")):
             full_path: str = os.path.join(source_path, file.name)
+            error_in_new_file = True
             convert_markers(full_path, target_path)
+            
+    print_error_stats()
 
 ################################################################################
 # Quick error printing logic
 ################################################################################
 error_location = ""
+error_in_new_file: bool = True
+error_total = 0
+files_with_errors = list()
+
 class ErrorType(Enum):
     INFO = 0
     WARNING = 1
     ERROR = 2
     
-def eprint(*args):
-    error_level = ErrorType.WARNING
-    for arg in list(args):
-        if(type(arg) == ErrorType):
-            error_level = arg
-            
-    print(error_level.name + " in file \"" + error_location + "\": ")
+def eprint(*args, error_level = ErrorType.WARNING):
+    global error_in_new_file
+    global error_total
+    error_total += 1
+    
+    if error_in_new_file:
+        print(error_level.name + " in file \"" + error_location + "\": ")
```

Comment:
Now that you are only printing out this line once per file you are only getting the error level of the first error message. If a warning is printed followed by an info or an error then the fact that the other messages are info or error will be hidden.